aspnetcore/blazor/get-started.md

```diff
@@ -84,51 +84,52 @@ To get started with Blazor, follow the guidance for your choice of tooling:
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
-Blazor Server is supported in Visual Studio for Mac. Blazor WebAssembly isn't supported at this time. To create Blazor WebAssembly apps on macOS, follow the guidance on the **.NET Core CLI** tab. For information on the two Blazor hosting models, *Blazor WebAssembly* and *Blazor Server*, see <xref:blazor/hosting-models>.
+> [!NOTE]
+> Blazor Server is supported in Visual Studio for Mac. Blazor WebAssembly isn't supported at this time, but a Blazor WebAssembly app can be created with the preview version of Visual Studio for Mac.
```

Comment:
Can we change this to something more similar to:

Blazor Server is supported in Visual Studio for Mac 8.4+, and Blazor WebAssembly support is included with Visual Studio for Mac 8.6+.